io.github.Torrent-file-editor.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>torrent-file-editor.desktop</id>
```

Comment:
```suggestion
  <id>torrent-file-editor</id>
```

Reply:
No. This should be the actuall app-id. There is no point changing a wrong to another wrong.

Reply:
fixed
https://github.com/flathub/flathub/pull/3457/files#diff-1ec4941b291ac3dc01816e41751d303a4b244014c2a9aef4b4d45df1cdc37290R9